exporters/stdout/stdoutmetric/config.go

```diff
@@ -74,6 +83,24 @@ func WithEncoder(encoder Encoder) Option {
 	})
 }
 
+// WithWriter sets the export stream destination.
+// If `WithEncoder` is also used, this option will have no effect.
+func WithWriter(w io.Writer) Option {
+	return optionFunc(func(c config) config {
+		c.writer = w
+		return c
+	})
+}
```

Comment:
I'm not a fan of having configs that don't do anything if others are used.
What if as an alternative, this is a shortcut for calling something like:
`WithEncoder(json.NewEncoder(w))`?

The same thing could be done with PrettyPrint.

Reply:
That's an excellent idea for WithWriter.

WithPrettyPrint is harder, as it modifies the encoder and would therefore require a specific order of options.
I have changed it to always set the indentation if the encoder is a JSON one.